### PR TITLE
Rejoin an online game

### PR DESCRIPTION
**Pull request by Chroq**
## Description

Trello: https://trello.com/c/3W8vHuQw/4-2-as-norbert-i-want-to-play-online-with-philippe

The goal of this PR is to rejoin an existing game. When a game is chosen and the form is  submitted, we send a PATCH request to add the new player to the game.

Based on the branch **game_mechanism**.

Todo :
- [x] Change the current "Rejoin" form

image
_http://localhost:3000/_

image
_http://localhost:3000/board/online?id=aedf84c0-ba62-4ce6-8fc9-b169c3c33d55_

### CODE REVIEW

File: components/forms/OnlineRejoinForm.js [hunk 1]
```diff
@@ -2,45 +2,37 @@ import { Button, FormLabel, Input, Select } from "@chakra-ui/core";
...
       onSubmit={(values) => {
-        Router.push({
-          pathname: "/board",
-          query: {
-            id: values.gameId,
-            player2Nickname: values.player2Nickname,
-            online: true,
-          },
+        addSecondPlayer(values.secondPlayerNickname, values.gameId).then(
+          function () {
```
Comment (djhi) [inline components/forms/OnlineRejoinForm.js:17]: Using arrow function would make this more readable imo

File: components/forms/OnlineRejoinForm.js [hunk 2]
```diff
@@ -2,45 +2,37 @@ import { Button, FormLabel, Input, Select } from "@chakra-ui/core";
...
       initialValues={{ gameId: firstGameId, nickname }}
       onSubmit={(values) => {
-        Router.push({
-          pathname: "/board",
-          query: {
-            id: values.gameId,
-            player2Nickname: values.player2Nickname,
-            online: true,
-          },
+        addSecondPlayer(values.secondPlayerNickname, values.gameId).then(
```
Comment (djhi) [inline components/forms/OnlineRejoinForm.js:16]: It seems the signature does not match your call here

Comment (Chroq) [reply components/forms/OnlineRejoinForm.js:16]: Indeed, weird :s Fixed, thanks

File: components/forms/OnlineRejoinForm.js (hunk 1 above)
Comment (Chroq) [reply components/forms/OnlineRejoinForm.js]: Changed, thanks !

**Pull request merged**